docs/pages/guides/analyzing-bundles.mdx

```diff
@@ -10,77 +10,39 @@ Bundle performance varies for different platforms. For example, web browsers don
+### Using Atlas with `expo export`
 
-```json package.json
-{
-  "scripts": {
-    "analyze:web": "source-map-explorer 'dist/_expo/static/js/web/*.js' 'dist/_expo/static/js/web/*.js.map'",
-    "analyze:ios": "source-map-explorer 'dist/_expo/static/js/ios/*.js' 'dist/_expo/static/js/ios/*.js.map'",
-    "analyze:android": "source-map-explorer 'dist/_expo/static/js/android/*.js' 'dist/_expo/static/js/android/*.js.map'"
-  }
-}
-```
+You can also use Expo Atlas when generating a production bundle for your app or EAS Update. Atlas generates a **.expo/atlas.jsonl** file during export, which you can share and open without having access to the project.
 
-If you are using the SDK 50 `server` output for web, then use the following to map web bundles:
+<Terminal cmd={[
+  '# Export your app for all platforms',
+  '$ EXPO_UNSTABLE_ATLAS=true npx expo export',
+  '',
+  '# Open the generated Atlas file',
+  '$ npx expo-atlas .expo/atlas.jsonl',
+]} />
 
-<Terminal
-  cmd={[
-    "$ npx source-map-explorer 'dist/client/_expo/static/js/web/*.js' 'dist/client/_expo/static/js/web/*.js.map'",
-  ]}
-/>
-
-Web bundles are output to the **dist/client** subdirectory to prevent exposing server code to the client.
-
-</Step>
-
-<Step label="3">
-
-Export your production JavaScript bundle and include the `--source-maps` flag so that the source map explorer can read the source maps:
-
-<Terminal
-  cmd={[
-    '$ npx expo export --source-maps --platform web',
-    '# Native apps using Hermes can disable bytecode for analyzing the JavaScript bundle.',
-    '$ npx expo export --source-maps --platform ios --no-bytecode',
-  ]}
-/>
-
-This command shows the JavaScript bundle and source map paths in the output. In the next step, you will pass these paths to the source map explorer.
-
-> Avoid publishing source maps to production as they can cause both security issues and performance issues (a browser will download the large maps).
-
-</Step>
-
-<Step label="4">
-
-Run the script to analyze your bundle:
-
-<Terminal cmd={['$ npm run analyze:web']} />
-
-On running this command, you might see the following error:
-
-```text
-You must provide the URL of lib/mappings.wasm by calling SourceMapConsumer.initialize({ 'lib/mappings.wasm': ... }) before using SourceMapConsumer
-```
-
-This is probably due to a [known issue](https://github.com/danvk/source-map-explorer/issues/247) in `source-map-explorer` in Node.js 18 and above. To resolve this, set the environment variable `NODE_OPTIONS=--no-experimental-fetch` before running the analyze script.
-
-</Step>
-
-You might encounter a warning such as `Unable to map 809/13787 bytes (5.87%)`. This occurs because source maps often exclude bundler runtime definitions (for example, `__d(() => {}, [])`). This value is consistent and not a reason for concern.
+You can also specify the platforms you want to analyze, using the `--platform` flag. Atlas will gather the data for the exported platforms only.
```

Comment:
```suggestion
You can also specify the platforms you want to analyze using the `--platform` option. Atlas will gather the data for the exported platforms only.
```

In our CLI tools, we often call options instead of flag. Maybe for consistency we should use the same term here.

Also, does this `--platform` option/flag requires passing the platform name? If so, may be we should mention that or provide a list fo all platforms (as it is not obvious to someone using this tool initially).

Reply:
`--platform` requires a name, yes. But, if you omit it, we error and tell it needs a value:

<img width="280" alt="image" src="https://github.com/expo/expo/assets/1203991/7f07794e-7dcc-4c94-a273-9d236de92de0">

Also in the `--help` output, we put everything in there:

<img width="553" alt="image" src="https://github.com/expo/expo/assets/1203991/e67d6f12-1ef1-4f7a-94bb-a7d89138c124">

I do think these "path-finding errors" are good enough to get users unblocked. If people have use it once, they should know the option.

Reply:
Speaking of, could add the "allowed options" to the error message. I think that's a great addition. Right now, we only do that for bad input:

<img width="566" alt="image" src="https://github.com/expo/expo/assets/1203991/338fc984-7ef3-45e4-a59d-3ce8b1ffbf3e">